corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -220,9 +220,19 @@ cloudCare.CaseListView = Backbone.View.extend({
         }
     },
     render: function () {
-	    var self = this;
-	    self.el = $('<section />').attr("id", "case-list").addClass("span7");
-        var table = $("<table />").addClass("table table-striped datatable").css('clear', 'both').appendTo($(self.el));
+        var self = this,
+            $panelBody,
+            $panel;
+        self.el = $('<section />').attr("id", "case-list").addClass("col-sm-7");
+        self.$el = $(self.el);
+        $panel = $('<div class="panel panel-default"></div>')
+            .append('<div class="panel-heading">Cases</div>');
+
+        $panelBody = $('<div class="panel-body"></div>');
+        $panel.append($panelBody);
+        self.$el.append($panel);
+
+        var table = $("<table />").addClass("table table-striped table-hover datatable").css('clear', 'both').appendTo($panelBody);
```

Comment:
no, was just taking it from the existing implementation. clearfix sounds like it'd be better though